include/new/trainer_sliding_data.h

```diff
@@ -14,14 +14,99 @@ extern const u8 BattleScript_TrainerSlideMsgEnd2[];
 extern const u8 gText_DefaultTrainerDynamaxMsg[];
 extern const u8 gText_TestTrainerDynamaxMsg[];
 
-//For Unbound
-
-extern const u8 sText_MirskleFirstMonDown[];
-extern const u8 sText_VegaLastSwitchIn[];
-extern const u8 sText_AliceLastSwitchIn[];
-extern const u8 sText_MelLastSwitchIn[];
-extern const u8 sText_GalavanLastSwitchIn[];
-extern const u8 sText_TessyLastSwitchIn[];
-extern const u8 sText_Zeph1LowHP[];
-extern const u8 sText_Zeph2LowHP[];
-extern const u8 sText_MaximaLastSwitchIn[];
+// Rival battle 1
+extern const u8 sText_Rival1_LastSwitchIn[];
+extern const u8 sText_Rival1_LowHP[];
+
+// Rival battle n
```

Comment:
why is this comment here? 
// Rival battle n

Reply:
It's a placeholder for now, until we know how many rival battles we'll have